cEP-0016.md

```diff
@@ -0,0 +1,178 @@
+Git Commit Content Inspection
+=============================
+
+|Metadata      |                                            |
+|--------------|--------------------------------------------|
+| cEP          | 16                                         |
+| Version      | 1.0                                        |
+| Title        | Git commit inspection                      |
+| Authors      | Kriti Rohillla <kritirohilla567@gmail.com> |
+| Status       | Implementation due                         |
+| Type         | Process                                    |
+
+## Abstract
+
+This document describes coala process rules which will be enforced by the
+GitCommitBear.
+
+This cEP describes some new changes in the implementation of GitCommitBear
```

Comment:
Why do we have "this document describes" twice? It sounds weird while reading it. Can't you just write it in one paragraph?

Like "This document describes coala process rules which will be enforced by the GitCommitBear and some new changes in its implementation.........."